dask_sql/physical/rex/core/call.py

```diff
@@ -596,6 +596,16 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format="%Y-%m-%d %H:%M:%S"):
+        df = df.astype("datetime64[s]")
+        df = df.dt.strftime(format)
```

Comment:
In addition to basic functionality on columns, PySpark allows an [optional `format` input](https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.functions.to_timestamp.html). Right now, I'm having trouble parsing an additional string input.

Reply:
Hi @beckernick I was wondering what the desired input for the `format` parameter would look like? For reference, these are the variations I've tried and the errors generated on the DataFusion side:

`SELECT to_timestamp(a, format="%Y-%m-%d %H:%M:%S") AS date FROM df` gives a `ParsingException: SchemaError(FieldNotFound { qualifier: None, name: "format", valid_fields: Some(["df.a"]) })`

`SELECT to_timestamp(a, "%Y-%m-%d %H:%M:%S") AS date FROM df` gives a `ParsingException: SchemaError(FieldNotFound { qualifier: None, name: "%Y-%m-%d %H:%M:%S", valid_fields: Some(["df.a"]) })`

`SELECT to_timestamp(a, '%Y-%m-%d %H:%M:%S') AS date FROM df` gives a `ParsingException: Plan("Coercion from [Int64, Utf8] to the signature Uniform(1, [Int64, Timestamp(Nanosecond, None), Timestamp(Microsecond, None), Timestamp(Millisecond, None), Timestamp(Second, None), Utf8]) failed.")`

`SELECT to_timestamp(a, %Y-%m-%d %H:%M:%S) AS date FROM df` gives a `ParsingException: ParserError("Expected an expression:, found: %")`